src/points.jl

```diff
@@ -13,8 +13,9 @@ immutable LLA{T <: Number}
     lon::T
     alt::T
 end
+LLA(lat, lon, alt=zero(lat)) = LLA(promote(lat, lon, alt)...)
 LLA(;lat=NaN,lon=NaN,alt=0.0) = LLA(lat,lon,alt) # Constructor that is idependent of storage order
-LLA{T}(lat :: T, lon :: T) = LLA(lat, lon, zero(T))
```

Comment:
I did mean to remove this. It was added as part of #28 which the new constructor I added also takes care of. I could add this back in but it would strictly be for a slight optimization when `lat` and `lon` are of the same type.